README.md

```diff
@@ -1 +1,133 @@
- hello-microverse
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [hello-microverse] <a name="about-project"></a>
+
+**[hello-microverse]** is a Microverse mini project to get done with the git and Github workflow.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Used html and css.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">html</a></li>
+    <li><a href="">css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">Github</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Getting to know the double step saving process]** By using git we become familiar with the workflow of the DVCS.
+- **[Improve commit messages]** By using the workflow and keeping in mind persons that will need to review the code we write better commit messages.
+- **[Pull Request]** Finally learning how to use branches and pull requests.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Project can be easily pulled and viewed in any web browser and code editor of choice.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A web browser to access the live version or the cloned project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:flov3rh/hello-microverse.git
+```
+
+### Usage
+
+To run the project, execute the following command simply open the live version or the index.html file.
+
+### Run tests
+
+To run tests, run the following commands on the root directory of the project:
+
+<!--
+Example command:
+
+```sh
+   npx hint .
+   npx stylelint "**/*.{css,scss}"
+
+```
+--->
```

Comment:
- [ ] @flov3rh well done you did make changes as per asked by your previous code reviewer, but there is a small issue and that is your steps that need to follow for running your test are commented out. Kindly remove the comments in order to make your steps visible publicly.

```suggestion
Example command:

```sh
   npx hint .
   npx stylelint "**/*.{css,scss}"

```

**Please remove <!-- --> from your README for the test section.**